playbook/app/pb_kits/playbook/pb_title/_title.scss

```diff
@@ -33,4 +34,21 @@
   &[class*=_thin] {
     @include pb_title_thin;
   }
+
+  @each $size, $size_value in $breakpoints_grid {
```

Comment:
Next time, (not a blocker!) we could simplify this whole function a little, and remove the @if @else... 

```
@import "../tokens/titles";
@import "../tokens/colors";
@import "../tokens/screen_sizes";
@import './title_mixin';

[class^=pb_title_kit] {
  @include title_colors;

  &[class*=_1], &[class*=_2], &[class*=_3], &[class*=_4] {
    @include pb_title_bold;
  }

  &[class*=_1] { @include pb_title_1; }
  &[class*=_2] { @include pb_title_2; }
  &[class*=_3] { @include pb_title_3; }
  &[class*=_4] { @include pb_title_4; }

  &.dark { @include pb_title_dark; }
  &[class*=_thin] { @include pb_title_thin; }

  @each $size, $size_value in $breakpoints_grid {
    @each $title_size_value in [1, 2, 3, 4] {
      $min_size: map-get($size_value, "min");
      $max_size: map-get($size_value, "max");
      &[class*=_#{$size}_#{$title_size_value}] {
        @include break_on($min_size, $max_size) {
          @include pb_title_#{$title_size_value};
          @include title_colors;
          @if $title_size_value != 4 { @include pb_title_bold; }
        }
      }
    }
  }
}

```

In my snippet, I moved the @include title_colors and @include pb_title_bold to the outer level of the selector since they were common for all cases. Then, I combined the separate blocks for [class*=_1], [class*=_2], [class*=_3], and [class*=_4] into a single block to make it a little more DRY (again, not a deal breaker).  And now, we wouldn't need the if-else statements, because including the appropriate pb_title_X mixins based on the value of $title_size_value.